ext/bg/js/options.js

```diff
@@ -111,19 +111,31 @@ const profileOptionsVersionUpdates = [
     },
     (options) => {
         // Version 14 changes:
-        //  Changed template for Anki audio.
+        //  Changed template for Anki audio and tags.
         let fieldTemplates = options.anki.fieldTemplates;
         if (typeof fieldTemplates !== 'string') { return; }
 
-        const replacement = '{{#*inline "audio"}}\n    {{~#if definition.audioFileName~}}\n        [sound:{{definition.audioFileName}}]\n    {{~/if~}}\n{{/inline}}';
-        let replaced = false;
-        fieldTemplates = fieldTemplates.replace(/\{\{#\*inline "audio"\}\}\{\{\/inline\}\}/g, () => {
-            replaced = true;
-            return replacement;
-        });
-
-        if (!replaced) {
-            fieldTemplates += '\n\n' + replacement;
+        const replacements = [
+            [
+                '{{#*inline "audio"}}\n    {{~#if definition.audioFileName~}}\n        [sound:{{definition.audioFileName}}]\n    {{~/if~}}\n{{/inline}}',
```

Comment:
1. Flip the order of the `[replacement, pattern]` values, since it's kind of a `pattern => replacement` style.
2. Move `new RegExp(escapeRegExp(` and `), 'g')` inside the `for`.

Reply:
https://github.com/FooSoft/yomichan/pull/539/commits/cf0f4a6a576f01d75b8bcae87e1c2cb90554985d